libmscore/edit.cpp

```diff
@@ -1214,35 +1214,53 @@ void Score::cmdFlip()
             MScore::setError(NO_NOTE_SLUR_SELECTED);
             return;
             }
+
+      std::set<const Element*> alreadyFlippedElements;
+      auto flipOnce = [&alreadyFlippedElements](const Element* element, std::function<void()> flipFunction) -> void {
+            if (alreadyFlippedElements.count(element) == 0) {
+                  alreadyFlippedElements.insert(element);
+                  flipFunction();
+                  }
+            };
       for (Element* e : el) {
             if (e->isNote() || e->isStem() || e->isHook()) {
-                  Chord* chord;
-                  if (e->isNote())
-                        chord = toNote(e)->chord();
+                  Chord* chord = nullptr;
```

Comment:
Initialization is not needed, especially as you do not check for nullptr later on.